server/sonar-web/src/main/js/apps/about/components/AboutAppForSonarQubeDotCom.js

```diff
@@ -45,72 +46,97 @@ type Props = {
   vulnerabilities: number
 };
 
-export default function AboutAppForSonarQubeDotCom(props: Props) {
-  const { customText } = props;
+type State = {
+  loadingFont: boolean
+};
 
-  return (
-    <div id="about-page" className="about-page sqcom-about-page">
-      <div className="sqcom-about-page-entry">
-        <div className="about-page-container">
-          <div className="sqcom-about-page-intro">
-            <h1 className="big-spacer-bottom">
-              Continuous Code Quality<br />as a Service
-            </h1>
-            <a
-              className="button button-active"
-              href="https://about.sonarqube.com/get-started/"
-              target="_blank">
-              Get Started
-            </a>
-            {!props.currentUser.isLoggedIn &&
-              <Link to="/sessions/new" className="button big-spacer-left">
-                {translate('layout.login')}
-              </Link>}
-          </div>
+export default class AboutAppForSonarQubeDotCom extends React.PureComponent {
+  props: Props;
+  state: State;
+
+  constructor(props: Props) {
+    super(props);
+    this.state = { loadingFont: true };
+    WebFont.load({
```

Comment:
this works, however I wonder if we could avoid pulling an extra dependency by using webpack's built-in `require.ensure` function to asynchronously load files

Check any `routes.js` for example.

We could have a wrapper component that will call `require.ensure` to fetch the `AboutAppForSonarQubeDotCom` component in its `componentDidUpdate`, and then render it.